pylint_nautobot/string_field_blank_null.py

```diff
@@ -35,10 +37,13 @@ def visit_classdef(self, node: ClassDef):
             # We are only interested in assignments to a call
             if not isinstance(child_node.value, Call):
                 continue
+            # We are only interested in calls to these two models
+            if child_node.value.func.attrname not in ("TextField", "CharField"):
```

Comment:
Right now the check is going too wide generating false positives.

Do you have a full list of known class names we should check against?

As for custom classes, that would require a lot more work to detect, if you have some examples please put them into an issue.

Reply:
Fine with this as a short-term urgent fix; I was hoping you might know of some way that pylint lets us walk the inheritance hierarchy for a class e.g. `isinstance(TextField, CharField)` instead of string checks. :-)

Reply:
Yeah, that's probably doable, I'd want to have it developed outside of this PR though. I've already discovered a case that was not dealt with by the original code due to my testing against golden-config :)